lms/djangoapps/mobile_api/social_facebook/friends/views.py

```diff
@@ -0,0 +1,69 @@
+"""
+Views for friends info API
+"""
+
+from rest_framework import generics, status
+from rest_framework.response import Response
+from openedx.core.djangoapps.user_api.api.profile import preference_info
+from opaque_keys.edx.keys import CourseKey
+from student.models import CourseEnrollment
+from ...utils import mobile_view
+from ..utils import get_friends_from_facebook, get_linked_edx_accounts
+from lms.djangoapps.mobile_api.social_facebook.friends import serializers
+from django.conf import settings
+
+_FACEBOOK_API_VERSION = settings.FACEBOOK_API_VERSION
+
+
+@mobile_view()
+class FriendsInCourse(generics.ListAPIView):
+    """
+    **Use Case**
+
+        API endpoint that returns all the users friends that are in the course specified.
+        Note that only friends that allow their courses to be shared will be included.
+
+    **Example request**:
+
+        GET /api/mobile/v0.5/social/facebook/friends/course/<course_id>
+
+        where course_id is in the form of /edX/DemoX/Demo_Course
+
+    **Response Values**
+
+        {   "friends":
+                [{
+                    "name": "test",
+                    "id": "12345",
+                    "edX_id": "678910"
```

Comment:
The app does not need the edX id it only needs to be used internally. 
I removed it from the response.